_packages/napari-nibabel.html

```diff
@@ -9,4 +9,8 @@
 category: "data-visualisation"
 ---
 
-<p>A napari i/o plugin for neuroimaging formats (via nibabel/pydicom). Note that this project is in an early prototype stage and is missing many features.</p>
+<p>A napari I/O plugin for neuroimaging formats (via nibabel/pydicom).</p>
+
+<p><a href="https://napari.org/">napari</a> is a fast, interactive, multi-dimensional image viewer for Python. It's designed for browsing, annotating, and analyzing large multi-dimensional images.</p>
+
+<p>Note that this project is in an early prototype stage and is missing many features.</p>
```

Comment:
Maybe state "plugin" rather than "project" in the last sentence if the "early prototype" description is meant to apply to the plugin as opposed to Napari itself? Napari is also still in alpha stage, though, so I guess the description could apply to either!

Reply:
```suggestion
<p>Note that this plugin is in an early prototype stage and is missing many features.</p>
```